Defs/Ammo/Shell/100x695mmR.xml

```diff
@@ -88,7 +88,7 @@
 		<defName>Bullet_100x695mmRCannonShell_HEAT</defName>
 		<label>100x695mmR cannon shell (HEAT)</label>
 		<graphicData>
-			<texPath>Things/Projectile/Cannon/HEAT</texPath>
+			<texPath>Things/Projectile/Cannon/HEATFS</texPath>
```

Comment:
Is `Cannon/HEATFS` intended and not `Tank/HEATFS`?

Reply:
for projectile yes